virttest/qemu_storage.py

```diff
@@ -594,6 +594,7 @@ def _parse_options(self, params):
                 "extent_size_hint",
                 ("qcow2", "raw", "luks"),
             ),
+            "image_compression_type": (None, "compression_type", ("qcow2",)),
```

Comment:
Sorry, I should add this comment yesterday, I just forgot it.
What about adding image_compression_type  in shared/cfg/base.cfg, you can refer to image_extent_size_hint. You might add some comments about this param, like it's used for a qcow2 image only, and the supported types are zlib and zstd currently